generator/src/generator/namespace/switch.rs

```diff
@@ -808,7 +809,19 @@ fn emit_variable_size_switch_serialize(
             ext_params_arg_defs
         );
         out.indented(|out| {
-            serialize::emit_assert_for_switch_serialize(generator, switch, switch_expr_type, out);
+            //serialize::emit_assert_for_switch_serialize(generator, switch, switch_expr_type, out);
```

Comment:
Perhaps a comment explaining why this line is there, but commented? I do not really "get it".